src/routes/amt/powerCapabilities.test.ts

```diff
@@ -12,7 +13,7 @@ describe('Power Capabilities', () => {
   let endSpy: jest.SpyInstance
   let mqttSpy: jest.SpyInstance
   let powerCaps
-  let version
+  // let version
```

Comment:
remove?

Reply:
Yes, made it as constant in test/help/wsmanresponse, so that I could use it in other tests too.

Reply:
Yes, i mean remove it :)